components/core/src/fs.rs

```diff
@@ -44,6 +44,17 @@ const SVC_PATH: &'static str = "hab/svc";
 lazy_static! {
     static ref EUID: u32 = users::get_effective_uid();
 
+    static ref MY_CACHE_PATH: PathBuf = {
```

Comment:
Yeah - if the user elect's to not create an origin key we won't have a key file/path.  So we needed a different check, and I elected to check the root path existence (could also have used the analytics path since the analytics opt in/out files are always created, but this seemed cleaner long term).